api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -3399,6 +3405,12 @@
     <concept_name  concept_name_id="160228"  concept_id="165180"  name="HIV First Time Tester"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-10 15:49:47"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="9bad469f-fecd-43cc-b84c-448e24695afc"/>
     <concept_name  concept_name_id="160227"  concept_id="165179"  name="Ward"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-10 15:45:02"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="362b39bf-55ff-4974-844d-97bc93dcfdca"/>
 
+    <!--PERSONAL ATTRIBUTE CONCEPTS NAMES-->
+    <concept_name  concept_name_id="160234"  concept_id="165185"  name="Foreigner"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-19 09:11:49"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="f01e3c72-7cdf-449e-9d59-ba5a408f163d"/>
+    <concept_name  concept_name_id="160233"  concept_id="165184"  name="National"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-19 09:10:47"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="6ade7b18-93ca-47a8-ba74-fdf497c93c96"/>
+    <concept_name  concept_name_id="160235"  concept_id="165186"  name="Nationality"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-19 09:13:57"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="95bfc486-e945-4fae-9cd7-fec9c0f44fd1"/>
```

Comment:
@solemabrothers Did you check if the concepts are already available in CIEL?

Reply:
Yes and they were non-existent